src/model/client.ts

```diff
@@ -70,15 +68,17 @@ export class CommandoClient extends Client {
     if (typeof filePath === 'string') {
       return walk(filePath).then(files => {
         files.forEach(async (file: string) => {
-          await this.resolveCommand(`${filePath}${filePath.endsWith('\\') ? '' : '\\'}${file}`)
+          await this.resolveCommand(
+            `${filePath}${filePath.endsWith('\\') || filePath.endsWith('/') ? '' : '/'}${file}`
```

Comment:
I think a more eloquent solution would be to use `path.join(filePath, file)` on line 72 and 81.